controlsfx/src/main/java/org/controlsfx/control/Notifications.java

```diff
@@ -228,6 +222,14 @@ public Notifications darkStyle() {
         return this;
     }
 
+    /**
+     * Specify the style classes to add to the {@link NotificationBar}.
```

Comment:
As per Oracle recommendation: "Use 3rd person (descriptive) not 2nd person (prescriptive)." -> [source](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html)

So better to go for "Specifies" I guess